mantidproject/mantid
# IDR Diagnostics should not restrict spectra numbers

Fixes #13525

To test:
- Open Indirect > Data Reduction > ISIS Diagnostics
- Instrument: OSIRIS
- Load run: osi89757.raw (from system test data)
- Set `Spectra Min` to 1
- Set `Preview Spectrum` to 1
- See that the plot shows the monitor spectrum

Release notes updates [here](http://www.mantidproject.org/index.php?title=Release_Notes_3_5_Indirect_Inelastic&diff=25103&oldid=25093).
